File: TugasPraktikum03/Bahasa Python/PRAK304-2310817320013-AikoAnatashaWendiono.py
angka = int(input("Input:\n"))

if (angka == 0):
    print("Output:\nNol")
elif (1 <= angka <= 9):
    print("Output:\nSatuan")
elif (10 <= angka <= 19):
    print("Output:\nBelasan")
elif (20 <= angka <= 99):
    print("Output:\nPuluhan")
else :
    print("Output:\nAnda Menginput Melebihi Limit Bilangan")

File: TugasPraktikum03/Bahasa C/PRAK304-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main() {
    int angka;

    printf("Input:\n");
    scanf("%d", &angka);

    if (angka == 0) {
        printf("Output:\nNol\n");
    } else if (1 <= angka && angka <= 9) {
        printf("Output:\nSatuan\n");
    } else if (10 <= angka && angka <= 19) {
        printf("Output:\nBelasan\n");
    } else if (20 <= angka && angka <= 99) {
        printf("Output:\nPuluhan\n");
    } else {
        printf("Output:\nAnda Menginput Melebihi Limit Bilangan\n");
    }

    return 0;
}
